api/src/main/java/org/openmrs/util/OpenmrsClassLoader.java

```diff
@@ -593,7 +593,6 @@ protected static void nullInstance(Object instance) {
 				field.setAccessible(true);
 				if (Modifier.isStatic(mods) && Modifier.isFinal(mods)) {
 					// Doing something recursively is too risky
-					continue;
```

Comment:
not sure this improves readability of the code if we simply remove this `continue` here because and leave the if/else block unchanged.

Reply:
@teleivo ok I will undo the changes. Thanks

Reply:
@teleivo I modified the code accordingly. Feel free to review. Thank you very much for your comments